src/Services.svelte

```diff
@@ -1,10 +1,9 @@
 <div class="py-24 my-16 bg-gray-200" id="services">
     <div class="max-w-screen-xl mx-auto px-4 sm:px-6 lg:px-8">
         <div class="lg:text-center">
-
-            <h3 class="mt-2 text-3xl leading-8 font-extrabold tracking-tight text-gray-700 sm:text-4xl sm:leading-10">
-                CUSTOM-SOFTWAREENTWICKLUNG
-            </h3>
+            <h2 class="text-3xl leading-9 tracking-tight font-extrabold text-gray-900 sm:text-4xl sm:leading-10 break-all">
```

Comment:
```suggestion
            <h2 class="text-3xl leading-9 tracking-tight font-extrabold text-gray-900 sm:text-4xl sm:leading-10 break-all uppercase">
```

this should keep it uppercase for the future when this text will be dynamic